lib/parse.ts

```diff
@@ -150,7 +150,7 @@ export const newsItem = ({
     modified,
     id: newsId,
     author: authorDisplayName,
-    intro: preamble,
+    intro: preamble.replace(/(?<=[,.!])(\w)/ig, ' $1'),
```

Comment:
How long do we expect preamble to be? I can see how this global expression might become quite expensive.

Reply:
I haven't seen any long ones and I change the regex in #102. Let's address it if it becomes a problem.